lib/aptible/cli/subcommands/log_drain.rb

```diff
@@ -0,0 +1,183 @@
+module Aptible
+  module CLI
+    module Subcommands
+      module LogDrain
+        def self.included(thor)
+          thor.class_eval do
+            include Helpers::Token
+            include Helpers::Database
+            include Helpers::LogDrain
+
+            desc 'log_drain:list', 'List all Log Drains'
+            option :environment
+            define_method 'log_drain:list' do
+              Formatter.render(Renderer.current) do |root|
+                root.keyed_list('description') do |node|
+                  scoped_environments(options).each do |account|
+                    account.log_drains.each do |drain|
+                      node.object do |n|
+                        ResourceFormatter.inject_log_drain(n, drain, account)
+                      end
+                    end
+                  end
+                end
+              end
+            end
+
+            desc 'log_drain:create:elasticsearch HANDLE '\
+                 '--db DATABASE_HANDLE --environment ENVIRONMENT',
+                 'Create an Elasticsearch Log Drain'
+            option :db, type: :string
+            option :environment
+            option :pipeline, type: :string
+            option :drain_apps, default: true, type: :boolean
+            option :drain_databases, default: true, type: :boolean
+            option :drain_ephemeral_sessions, default: true, type: :boolean
+            option :drain_proxies, default: true, type: :boolean
+            define_method 'log_drain:create:elasticsearch' do |handle|
+              account = ensure_environment(options)
+              database = ensure_database(options)
```

Comment:
Same as for the influxdb metric drain, this is enforcing that that the destination database must be in the same environment that you're creating the drain, let's not forget to at least add a warning/tip in the documentation for this restriction.